gradle.properties

```diff
@@ -47,7 +47,7 @@ typesafe_config_version=1.3.1
 apache_version=4.1.4
 apache_core_version=4.4.13
 gson_version=2.8.6
-okhttp_version=3.14.2
```

Comment:
Do we know, what minimal Android API level change does it introduce?

Reply:
```
The OkHttp 3.12.x branch supports Android 2.3+ (API level 9+) and Java 7+. 
OkHttp works on Android 5.0+ (API level 21+) and on Java 8+.
```
https://square.github.io/okhttp/
so it stays the same